tests/test_input_output.py

```diff
@@ -86,39 +86,41 @@ def test_np_environ(self):
         self.assertEqual(
             os.environ["DESC_BACKEND"],
             "numpy",
-            "numpy " "environment variable incorrect on use",
+            "numpy environment variable incorrect on use",
         )
 
     def test_quiet_verbose(self):
         ir = InputReader(self.argv2)
         self.assertEqual(
             ir.inputs[0]["verbose"],
             1,
-            "value of inputs['verbose'] " "incorrect on no arguments",
+            "value of inputs['verbose'] incorrect on no arguments",
         )
         argv = self.argv2 + ["-v"]
         ir = InputReader(argv)
         self.assertEqual(
             ir.inputs[0]["verbose"],
             2,
-            "value of inputs['verbose'] " "incorrect on verbose argument",
+            "value of inputs['verbose'] incorrect on verbose argument",
         )
         argv = self.argv2 + ["-vv"]
         ir = InputReader(argv)
         self.assertEqual(
             ir.inputs[0]["verbose"],
             3,
-            "value of inputs['verbose'] " "incorrect on double verbose argument",
+            "value of inputs['verbose'] incorrect on double verbose argument",
         )
         argv = self.argv2 + ["-q"]
         ir = InputReader(argv)
         self.assertEqual(
             ir.inputs[0]["verbose"],
             0,
-            "value of inputs['verbose'] " "incorrect on quiet argument",
+            "value of inputs['verbose'] incorrect on quiet argument",
         )
 
     def test_vmec_to_desc_input(self):
+        # FIXME: maybe just store a file we know is converted correctly,
```

Comment:
#262 this PR should add a test for this, can be simple and just reuse like the `HELIOTRON.nc` file to convert it